cmd/neofs-cli/modules/object/lock.go

```diff
@@ -104,7 +104,7 @@ func initCommandObjectLock() {
 	ff.StringSlice(commonflags.OIDFlag, nil, commonflags.OIDFlagUsage)
 	_ = objectLockCmd.MarkFlagRequired(commonflags.OIDFlag)
 
-	ff.Uint64P(commonflags.ExpireAt, "e", 0, "Lock expiration epoch")
+	ff.Uint64P(commonflags.ExpireAt, "e", 0, "Lock expires after that epoch")
```

Comment:
Can we just write `The last active epoch of the lock`, I mean `expire-at` is somewhat self-descriptive, we could clarify further.

Reply:
Updated.